plugins/dune-rules/src/dune_rules.ml

```diff
@@ -14,38 +14,71 @@ end = struct
       Arg.(
         required
         & pos 0 (some string) None
-        & info [] ~doc:"Interface file containing Gospel specifications"
+        & info [] ~doc:"Interface file containing Gospel specifications."
             ~docv:"INTERFACE")
 
     let config_file =
       Arg.(
-        required
-        & pos 1 (some string) None
-        & info [] ~doc:"Configuration file for Ortac/QCheckSTM" ~docv:"CONFIG")
+        value
+        & opt (some string) None
+        & info [ "c"; "config" ]
+            ~doc:
+              "Configuration file for Ortac/QCheck-STM. Useful for generating \
+               multiple test files per module under test."
+            ~absent:
+              "Concatenation of INTERFACE without the extension and \
+               \"_config.ml\"."
```

Comment:
Given how the `~absent` field is rendered, I think I would prefer no initial capital and no ending period.

```suggestion
              "concatenation of INTERFACE without the extension and \
               \"_config.ml\""
```